app/core/experiments.js

```diff
@@ -0,0 +1,9 @@
+export function getRequestAQuoteGroup (user) {
+  const bucket = user.get('testGroupNumber') % 10
```

Comment:
Will users always have a test group number?  Do I need to ensure this is set?

Reply:
Yeah this will always be set for the users, but do we want to run this test across all users or just US users? In case of US-only users you might want to use testGroupNumberUS instead.

Reply:
👍 will confirm